tests/plotting/test_actor.py

```diff
@@ -125,7 +135,7 @@ def test_actor_position(actor):
     assert actor.position == position
 
 
-def test_actor_rotate_x(actor):
+def test_actor_rotate(actor):
```

Comment:
nits. The original name is good following the naming rule.
```suggestion
def test_actor_rotate_x(actor):
```